var/spack/repos/builtin/packages/buddy/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import platform
+
+
+class Buddy(AutotoolsPackage):
+    """A Binary Decision Diagram library."""
+
+    homepage = "https://sourceforge.net/projects/buddy/"
+    url      = "https://sourceforge.net/projects/buddy/files/buddy/BuDDy%202.4/buddy-2.4.tar.gz"
+    list_url = url
+    list_depth = 1
+
+    version('2.4', sha256='d3df80a6a669d9ae408cb46012ff17bd33d855529d20f3a7e563d0d913358836')
+
+    depends_on('m4',       type='build')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool',  type='build')
```

Comment:
This tarball already comes with a configure script, so these dependencies probably aren't necessary.

Reply:
fixed in https://github.com/spack/spack/pull/16166/commits/d1db50ac0ebf56cdc4df55067447a44c54499ac3